traffic_portal/app/src/common/modules/form/server/FormServerController.js

```diff
@@ -80,6 +80,9 @@ var FormServerController = function(server, $scope, $location, $state, $uibModal
 
     $scope.server = server;
 
+    $scope.server.ip6IsService = $scope.server.ip6IsService === undefined ? true : $scope.server.ip6IsService;
```

Comment:
this is a bit confusing. not sure what's going on there.

Reply:
was just trying to default it to true if it is undefined, otherwise leave it as is